posthog/api/team.py

```diff
@@ -183,9 +183,10 @@ def get_permissions(self) -> List:
         Special permissions handling for create requests as the organization is inferred from the current user.
         """
         base_permissions = [permission() for permission in self.permission_classes]
+
+        # Return early for non-actions (e.g. OPTIONS)
         if self.action:
-            # Return early for non-actions (e.g. OPTIONS)
-            if self.action == "create":
+            if self.action in ("create", "reset_token"):
```

Comment:
`reset_token` should be `TeamMemberStrictManagementPermission` and not `OrganizationAdminWritePermissions`, as project-specific admins should have the same edit capabilities as org admins.

Reply:
Thanks for clarifying this!